request-manager/request/manager.go

```diff
@@ -542,6 +542,24 @@ func (m *manager) getWithJc(requestId string) (proto.Request, error) {
 	}
 
 	req.JobChain = &jc
+
+	var params proto.CreateRequestParams
+	var rawParams []byte // raw job chains are stored as blobs in the db.
+	q = "SELECT request FROM raw_requests WHERE request_id = ?"
```

Comment:
How could we get this info without making an extra call to the database?

Reply:
I thought the next line was a call to the database?